src/eval-frame/components/reps/rep-utils/get-top-level-rep-summary.js

```diff
@@ -0,0 +1,33 @@
+import { getObjAtPath } from "./get-child-summaries";
+import { serializeForValueSummary } from "./value-summary-serializer";
+import { getErrorStackString } from "./get-error-stack-summary";
+import { getInitialDataTableSummary } from "./get-data-table-summary";
+import { isRowDf } from "./rep-type-chooser";
+
+import UserReps from "../user-reps-manager";
+
+export function getTopLevelRepSummary(rootObjName, path) {
+  const value = getObjAtPath(window[rootObjName], path);
+
+  if (UserReps.getUserRepIfAvailable(value)) {
+    return {
+      repType: "HTML_STRING",
+      htmlString: UserReps.getUserRepIfAvailable(value)
```

Comment:
Just an FYI – the images here don't render

<img width="593" alt="Screen Shot 2019-07-18 at 4 47 26 PM" src="https://user-images.githubusercontent.com/95735/61499377-d0c2a700-a97b-11e9-8142-ddaddb960b7d.png">

Reply:
this is a problem with the API mike used for the example -- if you look closely you'll see a little magnifying glass icon, so the images that are being returned are rendering, they're just not great images.

that said, i'm not sure whether we'll want to support image tags or not in the long run -- def a tracking pixel vector, perhaps other security risks?